File: Leetcode75/1432_Most_candies.py
class Solution(object):
    def kidsWithCandies(self, candies, extraCandies):
        """
        :type candies: List[int]
        :type extraCandies: int
        :rtype: List[bool]
        """
        mostCandy = max(candies)
        answer = []
        for i in range(len(candies)):
            if candies[i] + extraCandies >= mostCandy:
                answer.append(True)
            else:
                answer.append(False)
        return answer

File: Leetcode75/1432_Most_candies.java
import java.util.Collections; 
import java.util.ArrayList;
import java.util.List;

class Solution {
    public List<Boolean> kidsWithCandies(int[] candies, int extraCandies) {
        int maxCandy = candies[0];
        for (int i = 0 ; i < candies.length ; i ++) {
            maxCandy = Math.max(maxCandy, candies[i]);
        }

        ArrayList<Boolean> answer = new ArrayList<>();
        for (int i = 0 ; i < candies.length ; i ++) {
            if (candies[i] + extraCandies >= maxCandy) {
                answer.add(true);
            } else {
                answer.add(false);
            }
        }
        return answer;
    }
}
